packages/patternfly-4/react-core/src/components/Tooltip/Tooltip.tsx

```diff
@@ -53,23 +53,28 @@ export interface TooltipProps {
   isAppLauncher?: boolean;
   /** Maximum width of the tooltip (default 12.5rem) */
   maxWidth?: string;
-  /** 
-   * Tooltip position. Note: With 'enableFlip' set to true, 
+  /**
+   * Tooltip position. Note: With 'enableFlip' set to true,
    * it will change the position if there is not enough space for the starting position.
    * The behavior of where it flips to can be controlled through the flipBehavior prop.
    */
   position?: 'auto' | 'top' | 'bottom' | 'left' | 'right';
-  /** Tooltip trigger: click, mouseenter, focus */
+  /** Tooltip trigger: click, mouseenter, focus, manual  */
   trigger?: string;
+  /** value for visibility when trigger is 'manual' */
+  isVisible?: boolean;
   /** z-index of the tooltip */
   zIndex?: number;
+  /** additional Props to pass through to tippy.js */
+  tippyProps?: TippyProps;
```

Comment:
i think we should just add what we need in terms of props instead of a passthrough? otherwise could be more maintenance if some tippy props don't behave as expected in our component

Reply:
I think it's better to allow the full set to be passed thru. How can we know up front which props consumers want to customize? Limiting the options will cause limits to its adoptability. If there are any issues with using a particular prop, then an issue can be raised.

Reply:
what about the fact that there'll be duplicate props, for example we expose `enableFlip` which would be the same as `tippyProps.flip`

Reply:
whatever props we accept/set will override what is in the tippyProps

Reply:
okay let's go with this, can you also add the passthrough to the Popover component as well then? Thanks